app/models/planet.py

```diff
@@ -0,0 +1,20 @@
+class Planet:
+    def __init__(self, id, name, description, size_dia_km):
```

Comment:
👍 Great idea to include more specifics about what "size" means (the diameter in kilometers). This is often useful to do with a data value that's otherwise just a number.